docs/source/index.rst

```diff
@@ -1,53 +1,164 @@
+                    :color: primary
+
+                    MacOS M1 64-bit :octicon:`download`
+
+    .. tab-item:: Linux :fab:`linux`
+
+        Miniconda installer for:
+
+        .. grid:: 2
+
+            .. grid-item::
+
+                .. button-link:: https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+                    :color: primary
+
+                    Linux x86 64-bit :octicon:`download`
+
+    .. tab-item:: Homebrew :fa:`beer-mug-empty`
+
+        Run the following `Homebrew <https://brew.sh/>`_ command:
+
+        .. code-block:: bash
+
+            brew install miniconda
+
+    .. tab-item:: Chocolatey :octicon:`terminal` :fab:`windows`
+
+        Run the following `Chocolatey <https://chocolatey.org/>`_ command:
+
+        .. code-block:: bash
+
+            choco install miniconda3
+
+
+.. raw:: html
+
+    <p class="text-small">For more detailed instructions, see <a href="https://docs.conda.io/projects/miniconda/en/latest/">Miniconda's installation guide</a></p>
+
+New to conda? :octicon:`rocket;1em;sd-text-primary`
+...................................................
+
+If you are new to conda, we first recommend the following articles:
+
+.. grid:: 1 2 2 2
+    :gutter: 2
+
+    .. grid-item-card:: Getting started guide :octicon:`rocket;1em;sd-text-primary`
+        :link: /user-guide/getting-started
+        :link-type: doc
+
+        Learn the basics of using conda such as creating and adding packages to environments
+
+    .. grid-item-card:: Managing environments :octicon:`file-submodule;1em;sd-text-primary`
+        :link: /user-guide/tasks/manage-environments
+        :link-type: doc
+
+        Go in depth about environments and best practices for using them in your projects
+
+.. seealso::
+
+    Want to get even more in-depth training on how to use conda for free? Check out Anaconda's
+    `free course on conda basics <https://learning.anaconda.cloud/conda-basics>`_.
+
+Other useful resources :octicon:`light-bulb;1em;sd-text-primary`
+......................................................................
+
+.. grid:: 1 2 2 2
+    :gutter: 2
+
+    .. grid-item-card:: Command reference :octicon:`terminal;1em;sd-text-primary`
+        :link: /commands/index
+        :link-type: doc
+
+        Full reference for all standard commands and options
+
+    .. grid-item-card:: Cheatsheets :octicon:`note;1em;sd-text-primary`
+        :link: /user-guide/cheatsheet
+        :link-type: doc
+
+        Get the latest cheatsheet for commonly used commands
+
+    .. grid-item-card:: Configuring conda :octicon:`gear;1em;sd-text-primary`
+        :link: /user-guide/configuration/use-condarc
+        :link-type: doc
+
+        Learn about the various ways conda's behavior can be configured
+
+    .. grid-item-card:: Glossary :octicon:`book;1em;sd-text-primary`
+        :link: /glossary
+        :link-type: doc
+
+        Important vocabulary to know when working with conda
```

Comment:
Let's not remove the trailing slash, that means another redirect for every click

```suggestion
    .. grid-item-card:: Command reference :octicon:`terminal;1em;sd-text-primary`
        :link: /commands/index/
        :link-type: doc

        Full reference for all standard commands and options

    .. grid-item-card:: Cheatsheets :octicon:`note;1em;sd-text-primary`
        :link: /user-guide/cheatsheet/
        :link-type: doc

        Get the latest cheatsheet for commonly used commands

    .. grid-item-card:: Configuring conda :octicon:`gear;1em;sd-text-primary`
        :link: /user-guide/configuration/use-condarc/
        :link-type: doc

        Learn about the various ways conda's behavior can be configured

    .. grid-item-card:: Glossary :octicon:`book;1em;sd-text-primary`
        :link: /glossary/
        :link-type: doc

        Important vocabulary to know when working with conda
```

Reply:
Sounds good.

Was unaware of this behavior.